test/unittests/commands/excel2json/test_properties.py

```diff
@@ -15,27 +13,15 @@
 from pandas.testing import assert_frame_equal
 
 from dsp_tools.commands.excel2json import properties as e2j
-from dsp_tools.models.exceptions import UserError
+from dsp_tools.models.exceptions import InputError
+from dsp_tools.models.input_error import InvalidExcelContentProblem
 
+excelfile = "testdata/excel2json/excel2json_files/test-name (test_label)/properties.xlsx"
+output_from_method, _ = e2j.excel2properties(excelfile, None)
```

Comment:
Perhaps not that important, but a while ago I learnt that there is a difference between function and method:
- function = standalone (without a class)
- method = inside a class

I did this naming wrong in many places in this codebase, because I didn't know the difference. But now that I learnt it, I thought I will spread the word :-)